src/sql/parts/notebook/models/modelInterfaces.ts

```diff
@@ -510,4 +511,5 @@ export interface ICellMagicMapper {
 
 export namespace notebookConstants {
 	export const SQL = 'SQL';
+	export const SQL_PROVIDER = 'MSSQL';
```

Comment:
can we put the word "connection" in the name somewhere? like SQL_CONNECTION_PROVIDER? Just confusing when there are notebook + connection providers.

Reply:
Renamed. Do you mind if I move the constant into a separate file